docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -359,7 +367,18 @@ Both `interval` and `lookback` have a format of `"{integer}{time_unit}"`, where
 
 |==============================================
 
-<<bulk-edit-object-schema, Actions>> are shown in order of oldest to newest in the `edit` array payload's property.
+[[bulk-manual-rule-run]]
+[discrete]
+==== BulkManualRuleRun
+
+* TBD
+
+[[bulk-edit-object-schema]]
+[discrete]
+==== BulkEditAction object
```

Comment:
Does this mean that we put `start_date` and `end_date` fields under `BulkEditAction`? Those fields should be under `BulkManualRuleRun`. There were no changes to `BulkEditAction` object.

Also, it looks like we removed actions section header which we still need
> <<bulk-edit-object-schema, Actions>> are shown in order of oldest to newest in the `edit` array payload's property.

Reply:
No, sorry this is a typo - I forgot to update the header for this new section.

Reply:
[cee1fcc](https://github.com/elastic/security-docs/pull/5589/commits/cee1fcc0d4991dc42fd17f73d0ace8db21e92bfa) re-adds the line that I accidentally deleted